gatsby-node.js

```diff
@@ -265,11 +265,15 @@ exports.createPages = async ({ actions, graphql, reporter }) => {
           i18nNode.fields.slug.replace(`/${locale}`, '') === node.fields.slug
       );
 
-      createPageFromNode(i18nNode || node, {
-        prefix: i18nNode ? '' : locale,
-        createPage,
-        disableSwiftype: !i18nNode,
-      });
+      createPageFromNode(
+        i18nNode || node,
+        {
+          prefix: i18nNode ? '' : locale,
+          createPage,
+          disableSwiftype: !i18nNode,
+        },
+        !!i18nNode // defer if localized
```

Comment:
i think we'd always want to defer these (localized pages w/english and localized pages with translated content). the check if it's an `i18nNode` in `disableSwiftype` is to ensure we're only indexing pages with translated content in the search engine.

Reply:
This makes sense and clarifies the logic here then. I can fix that 👍🏻